### ISSUE TITLE
Copy metaAssembly wdl file to this repo and replace the assembler spades with megahit for metaT.

### ISSUE THREAD

**Issue by mshakya** [microbiomedata/metaT_Assembly#1]
- metaAssembly repo: https://github.com/microbiomedata/metaAssembly
- WDL file to be copied: [jgi_assembly.wdl](https://github.com/microbiomedata/metaAssembly/blob/master/jgi_assembly.wdl)
- Only have tasks related to assembly in this repo, so task `read_mapping_pairs`  can be removed.
- Also megahit does not have a separate repo in dockerhub, so can just use this one for now: `vout/megahit:release-v1.2.9`
- Megahit parameter is based off this: https://code.jgi.doe.gov/BFoster/jgi_meta_wdl/-/tree/master/metatranscriptome. In nutshell, replace spades command with this:
```
# for paired reads
megahit -t ${spades_cpu} --k-list  23,43,63,83,103,123 -m 100000000000 -o ${outprefix} -1 ${infile1} -2 ${infile2}

# for single end reads
megahit -t ${spades_cpu} --k-list  23,43,63,83,103,123 -m 100000000000 -o ${outprefix} -r ${infile1}

```
- the current implementation of metaAssembly should handle generation of infile1 and infile2.

**Comment by aclum** [comment]
What are the blockers from importing a wdl from JGI's repo instead of copy?

**Comment by mshakya** [comment]
sorry, didnt see this earlier. Is there an example wdl that imports directly from JGI (annotation one maybe?) ? Assembly and ReadsQC are nmdc specific that emulates JGI wdl. One method might be just git clone the wdls or just have dockers for wdls.

**Comment by mshakya** [comment]
Also, i have focused on using spades now.

**Comment by aclum** [comment]
none of the nmdc workflows do this now, we'd have to update the workflows to wdl version 1.0, this is straightforward, to be able to use url imports. I dont think we can have dockers for the wdls b/c you cant run docker within docker.

**Comment by mshakya** [comment]
I tried direct import like this:
```
import "https://raw.githubusercontent.com/microsoft/five-dollar-genome-analysis-pipeline-azure/az1.1.0/structs/GermlineStructs.wdl" as http_rnaspades
``` 

based on cromwell's [HELPDOC](https://cromwell.readthedocs.io/en/stable/Imports/), but was not able to get it working as the wdl file was not being accessed and obtained. I tried this with version 1.0 and same error. 

I used git submodule instead, which i think also provides a cleaner alternative to copying.

Closing this issue for now.
